docs/fl_introduction.rst

```diff
@@ -0,0 +1,68 @@
+.. _fl_introduction:
+
+###########################
+What is Federated Learning?
+###########################
+
+Federated Learning is a distributed learning paradigm where training occurs across multiple clients, each with their own local datasets.
+This enables the creation of common robust models without sharing sensitive local data, helping solve issues of data privacy and security.
+
+How does Federated Learning Work?
+=================================
+The federated learning (FL) aggregrator orchestrates the collaboration of multiple clients by first sending an initial model to the FL clients.
+The clients perform training on their local datasets, then send the model updates back to the FL aggregrator for aggregation to form a global model.
+This process forms a single round of federated learning and after a number of rounds, a robust global model can be developed.
+
+.. note::
+
+    In the diagrams below, the FL server has the role of the FL aggregrator. In the case of client-controlled workflows such as swarm learning,
+    FL clients can serve as FL aggregrators instead.
+
+.. image:: resources/fl_diagram.png
+    :height: 500px
+    :align: center
+
+FL Terms and Definitions
+========================
+
+- FL server: manages job lifecycle
+- FL aggregrator: orchestrates workflow, assigns tasks to clients, performs aggregation
```

Comment:
typo? aggregrator -> aggregator?

Also just a note this concept of FL aggregator is broader than our "Aggregator" class, since our aggregator classes only do aggregation.

Reply:
Right, technically it would be more like "controller", but since this is the overview I didn't want to go into detail for flare specific terms